modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js

```diff
@@ -140,6 +206,25 @@ const syncActions = (pages, actions) => {
 				.filter((key) => !targetFieldExists(outputs[key], pages))
 				.map((key) => delete outputs[key]);
 		}
+		else if (action.action === 'calculate') {
+			const expressionFields = getExpressionFields(action);
+
+			if (expressionFields && expressionFields.length > 0) {
+				expressionFields.forEach((field) => {
+					if (!targetFieldExists(field, pages)) {
+						const inexistentField = new RegExp(field, 'g');
```

Comment:
@alinedoleron, please add a break line after this line. Ideally, we should separate variable declarations from statements. 😉